tests/Integration/Db/CodeMapperTest.php

```diff
@@ -79,11 +79,9 @@ public function testExists() {
 		$this->assertTrue($exists);
 	}
 
-	/**
-	 * @expectedException \OCP\AppFramework\Db\DoesNotExistException
-	 */
-	public function testFindNotFound() {
-		$user = $this->createTestUser();
+    public function testFindNotFound() {
+        $this->expectException(\OCP\AppFramework\Db\DoesNotExistException::class);
+        $user = $this->createTestUser();
```

Comment:
indentation is off

Reply:
Uppsie, thanks for spotting this. Turned out that tab usage was not turned on for this project in my IDE. Fixed now and all PRs force-pushed :blush: